tests/context/context_constructors.cpp

```diff
@@ -128,9 +128,13 @@ class TEST_NAME : public util::test_base {
           FAIL(log, "context was not copied correctly (is_host)");
         }
 
-#ifdef SYCL_CTS_TEST_OPENCL_INTEROP
-        if (!selector.is_host() && (contextA.get() != contextB.get())) {
-          FAIL(log, "context was not copied correctly (get)");
+#ifdef SYCL_BACKEND_OPENCL
+        auto queue = util::get_cts_object::queue();
+        if (queue.get_backend() == sycl::backend::opencl) {
+          if (sycl::get_native<sycl::backend::opencl>(contextA) !=
+              sycl::get_native<sycl::backend::opencl>(contextB)) {
+            FAIL(log, "context was not copied correctly (get)");
```

Comment:
```suggestion
            FAIL(log, "context was not copied correctly");
```
What is the point of `(get)` (or `(get_native)`) part of the error message? There is no such part in tests/event/event_constructors.cpp.
I suggest we apply the same style to all similar messages and remove ` (get)` part of the message.